transform/esios_transform.py

```
@@ -427,27 +427,27 @@ def _process_latest_day(self, mercado: str) -> Optional[pd.DataFrame]:
 
         return processed_df # Return the result (DF or None)
 
-    def _process_date_range(self, mercado: str, start_date: str, end_date: str) -> Optional[pd.DataFrame]:
+    def _process_date_range(self, mercado: str, fecha_inicio: str, fecha_fin: str) -> Optional[pd.DataFrame]:
         """Processes data for a specified date range and returns a single combined processed DataFrame."""
         print("\n" + "="*80)
         print(f"🔄 STARTING MULTIPLE DAY TRANSFORM (Date Range)")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)